docs/CHANGELOG.md

```diff
@@ -20,6 +20,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Support Bugzilla tracker creation/linking for non-Bugzilla flaws (OSIDB-2845)
 - Add bulk-enabling parameter "sync_to_bz" to POST for Trackers (OSIDB-2609)
 - Add bulk POST, DELETE for Affects (OSIDB-2722)
+- Add major incident start date field (OSIDB-2728)
```

Comment:
I would mention the whole name of a field for transparency, i.e. Add "major_incident_start_dt" field to Flaw